gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ConfigBasedDatasetsFinder.java

```diff
@@ -50,36 +63,43 @@
  */
 
 @Slf4j
-public class ConfigBasedDatasetsFinder implements DatasetsFinder<ConfigBasedDataset> {
+public class ConfigBasedDatasetsFinder implements DatasetsFinder<CopyableDatasetBase> {
+
+  //prefix key for config store used by data replication
+  public static final String GOBBLIN_REPLICATION = "gobblin.replication";
 
   // prefix key for config store used by data replication
-  public static final String GOBBLIN_REPLICATION_CONFIG_STORE = "gobblin.replication.configStore";
+  public static final String GOBBLIN_REPLICATION_CONFIG_STORE = GOBBLIN_REPLICATION + ".configStore";
```

Comment:
Can you use `ConfigurationKeys.CONFIG_MANAGEMENT_STORE_URI` instead (used by other flows)